packages/vue-app/template/App.js

```diff
@@ -98,7 +98,18 @@ export default {
   },
   created () {
     // Add this.$nuxt in child instances
-    Vue.prototype.<%= globals.nuxt %> = this
+    if (process.server) {
+      this.$root.$options.<%= globals.nuxt %> = this
+      Object.defineProperty(Vue.prototype, '<%= globals.nuxt %>', {
+        get() {
+          return this.$root.$options.<%= globals.nuxt %>
+        },
+        configurable: true
+      })
+    } else {
+      Vue.prototype.<%= globals.nuxt %> = this
```

Comment:
Why different behavior for client-side?

Reply:
I think if we're moving into global context - which is better - then we don't need different behaviour because we won't be reassigning the property.